### PR TITLE
Quick Ubuntu install instructions for Readme

### PR DESCRIPTION
**Pull request by patrykkalinowski**
In my opinion some quick instruction for less tech-savy users would be helpful.

As a rookie Linux user I was struggling myself with Redshift not working when launched - I couldn't see that it could not determine my geo location until I started working in terminal on the daily basis - and that's when I've seen log output explaining everything.

### CODE REVIEW

**Comment by TingPing** [comment]
1. Only 3 words of that are Ubuntu related
2. Probably should instruct them to use the config file.

**Comment by eosrei** [comment]
This would have absolutely saved me a few minutes when I first installed.

**Comment by simonnagl** [comment]
In my opinion distribution related install-guides should be on the dirstribution related wikis/forums.
For Ubuntu this is https://wiki.ubuntuusers.de/Redshift/.

File: README.md [hunk 1]
```diff
@@ -24,6 +24,18 @@ Building from source
 
 See the file [HACKING](HACKING.md) for more details on building from source.
 
+Quick install & forget on Ubuntu
+-----------------------
+Download Redshift using Ubuntu Software Center or launch command in terminal: `sudo apt-get install redshift-gtk`
+
+Open system menu and launch Session and Startup. Redshift should be already on the list. Tick the box next to it's name. From the next login, Redshift will adjust your screen automatically.
```
Comment (Kasijjuf) [inline README.md]: What system menu do you mean?

Comment (m1cm1c) [reply README.md]: @Kasijjuf I think he uses Xfce. I don't see anything titled "Session and Startup" in the system settings of vanilla Ubuntu 16.04.

**Comment by jonls** [comment]
The current README has been updated with examples on how to install the package on Ubuntu. The other information seems to be Xfce-specific. It's difficult to keep this kind of information up to date so I'll leave it to the specific desktops to document this for users.

**Pull request closed without merging**